internal/common/common.go

```diff
@@ -143,6 +144,32 @@ func NormalizeCPUArchitecture(arch string) string {
 	}
 }
 
+// Return available disks and available disks that meet the minSize requirement
+func GetValidDiskCount(disks []*models.Disk, installationDiskID string, minSize int64) (int64, int64) {
+	var eligibleDisks int64
+	var availableDisks int64
+
+	for _, disk := range disks {
+		if (disk.DriveType == models.DriveTypeSSD || disk.DriveType == models.DriveTypeHDD) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
+			if disk.SizeBytes >= conversions.GbToBytes(minSize) {
+				eligibleDisks++
+			} else {
+				availableDisks++
+			}
+		}
+	}
+	return eligibleDisks, availableDisks
+}
```

Comment:
Maybe it should be in internal/operators/common.go?

Reply:
Cannot move it there because of circular dependency (assuming package would be `operators`)
To avoid this problem, I would suggest using `<package>utils` pattern for stateless functions that might be used outside the package. For example, I would move this function in `inventoryutils` package - as all needed data is inventory-related, and `HasOperator` function in `clusterutils` - as all necessary data is cluster-related. WDYT?

Reply:
These methods are used in a single package, so they shouldn't be in common/common.go, but rather in the package they are used.

Reply:
They are not a single package: "operators/lvm" and "operators/odf" are two separate packages. This said, I agree it shouldn't be in "common", but I'll move it to "operators/common" as moving to "operators" will cause circular dependency.